packages/protocol/src/protocol/message_layer/StreamMessage.ts

```diff
@@ -1,18 +1,16 @@
 import InvalidJsonError from '../../errors/InvalidJsonError'
 import StreamMessageError from '../../errors/StreamMessageError'
 import ValidationError from '../../errors/ValidationError'
-import UnsupportedVersionError from '../../errors/UnsupportedVersionError'
 import { validateIsNotEmptyString, validateIsString, validateIsType } from '../../utils/validations'
 
 import MessageRef from './MessageRef'
 import MessageID from './MessageID'
 import EncryptedGroupKey from './EncryptedGroupKey'
-import { Serializer } from '../../Serializer'
 import { StreamID } from '../../utils/StreamID'
 import { StreamPartID } from '../../utils/StreamPartID'
 import { EthereumAddress } from '@streamr/utils'
+import { fromArray, toArray } from './streamMessageSerialization'
 
-const serializerByVersion: Record<string, Serializer<StreamMessage>> = {}
```

Comment:
Maybe now we could have only one place where the version number `32` is defined.

E.g. static LATEST_VERSION = 32 at line 54 and use that static variable in `streamMessageSerialization.ts#8` and `streamMessageSerialization.test.ts#11`. Or alternatively we could hide the static version number field from public API?

Reply:
Feel free to add commit with this change